src/configureWorkspace/scaffolding.ts

```diff
@@ -36,6 +36,7 @@ export interface ScaffolderContext extends ScaffoldContext {
     platform: Platform;
     promptForOS(): Promise<PlatformOS>;
     promptForPorts(defaultPorts?: number[]): Promise<number[]>;
+    promptForCompose(): Promise<boolean>;
```

Comment:
Update generic-configure-scaffolding and Python scaffolder to use this new function?